lib/resources/tasks/jest.js

```diff
@@ -13,9 +13,9 @@ export default (cb) => {
 
   process.env.BABEL_TARGET = 'node';
 
-  jest.runCLI(options, [path.resolve(__dirname, '../../')]).then((result) => {
-    if (result.numFailedTests || result.numFailedTestSuites) {
-      cb(new PluginError('gulp-jest', { message: 'Tests Failed' }));
+  jest.runCLI(options, [path.resolve(__dirname, '../../')]).then(({ results }) => {
+    if(results.numFailedTests || results.numFailedTestSuites) {
```

Comment:
Could you leave a space between `if` and `(` for both jest.js and jest.ts? I see it was not there in jest.ts before, `if(` looks weird without space.

Reply:
@huochunpeng Updated the PR. Seems like it was already present in the TS file, I guess I just copy pasted it and it did not auto format. (We could use a prettier config in the cli repo or even better a prettier pre-commit hook)